src/functionalTest/resources/application-functional.yaml

```diff
@@ -1,8 +1,6 @@
-targetInstance: ${TEST_URL:http://rd-professional-api-aat.service.core-compute-aat.internal}
+targetInstance: ${TEST_URL:https://rd-professional-api-pr-233.service.core-compute-preview.internal}
```

Comment:
This is NOT going to work for other pr's except 233!  Please revise to either discover what the pr- is dyncamically from the environment somehow; or use a universal test env like demo.

Reply:
This change not to move demo until shared case push to production so that is the reason tetsing in PR branch rather than demo.